go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	golang.org/x/crypto v0.21.0
 	golang.org/x/net v0.21.0
 	golang.org/x/oauth2 v0.17.0
+	golang.org/x/text v0.14.0
```

Comment:
Why import this?

Reply:
This is to encode the password as UTF-16, because in Microsoft AD domains, passwords are required to be encoded in UTF-16. If you design your own function to implement the encoding, it may not be as stable and could lead to unexpected errors.